src/odemis/gui/cont/project.py

```diff
@@ -215,7 +221,7 @@ def _add_roa_ctrl(self, coords):
         # Enable buttons of project bar
         self._project_bar.enable_buttons(True)
 
-    # Should be called from GUI main thread
+    # already running in main GUI thread as it receives event from GUI
```

Comment:
Well, it's not directly receiving events from the GUI. Looking at the code, I see it's called from `_on_btn_remove()`, which is receiving a GUI event. I'd suggest to add a docstring saying "Must be called from within the main GUI thread".